exchanges/coinbasepro/coinbasepro.go

```diff
@@ -150,15 +149,11 @@ var (
 	errUnknownL2DataType        = errors.New("unknown l2update data type")
 	errOrderFailedToCancel      = errors.New("failed to cancel order")
 	errUnrecognisedStatusType   = errors.New("unrecognised status type")
-	errStringConvert            = errors.New("unable to convert into string value")
-	errFloatConvert             = errors.New("unable to convert into float64 value")
 	errWrappedAssetEmpty        = errors.New("wrapped asset cannot be empty")
-	errUnrecognisedOrderType    = errors.New("unrecognised order type")
-	errUnrecognisedAssetType    = errors.New("unrecognised asset type")
 	errUnrecognisedStrategyType = errors.New("unrecognised strategy type")
-	errIntervalNotSupported     = errors.New("interval not supported")
 	errEndpointPathInvalid      = errors.New("endpoint path invalid, should start with https://")
 	errPairsDisabledOrErrored   = errors.New("pairs are either disabled or errored")
+	errTypeAssert               = errors.New("type assertion failed")
```

Comment:
When `common.GetTypeAssertError` is called it returns `common.ErrTypeAssertFailure` so this can be removed